pocs/tests/test_observatory.py

```diff
@@ -27,7 +29,10 @@ def simulator():
 @pytest.fixture
 def observatory(config, simulator, images_dir):
     """Return a valid Observatory instance with a specific config."""
+    details = setup_site_location_details_from_config(config)
```

Comment:
See comments above but `details` is also a meaningless name as it could refer to the details of anything. This is `location_info` or `site_info`. Since it contains keys called `location` then it should probably be `site_details`.